frontend/packages/console-app/src/components/modals/resource-limits/ResourceLimitsModal.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import * as _ from 'lodash';
+import { FormikProps, FormikValues } from 'formik';
+import {
+  ModalTitle,
+  ModalBody,
+  ModalSubmitFooter,
+} from '@console/internal/components/factory/modal';
+import ResourceLimitSection from '@console/dev-console/src/components/import/advanced/ResourceLimitSection';
+
+interface ResourceLimitsModalProps {
+  cancel?: () => void;
+}
+
+type Props = FormikProps<FormikValues> & ResourceLimitsModalProps;
+
+const ResourceLimitsModal: React.FC<Props> = ({
+  handleSubmit,
+  cancel,
+  isSubmitting,
+  status,
+  errors,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <form className="modal-content modal-content--no-inner-scroll" onSubmit={handleSubmit}>
+      <ModalTitle>{t('modal~Edit resource limits')}</ModalTitle>
+      <ModalBody>
+        <ResourceLimitSection hideTitle />
+      </ModalBody>
+      <ModalSubmitFooter
+        submitDisabled={!_.isEmpty(errors)}
+        inProgress={isSubmitting}
+        errorMessage={status && status.submitError}
+        submitText={t('modal~Save')}
+        cancel={cancel}
```

Comment:
should we add translation for the button with cancel text as well?

Reply:
it's already translated
![rl](https://user-images.githubusercontent.com/22490998/114172123-e1d2f900-9952-11eb-9699-16fd85c2e634.png)